t/t2800-jobs-instance-info.t

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+test_description='Test flux jobs command with instance.* attributes'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4 job
+
+export FLUX_URI_RESOLVE_LOCAL=t
+waitfile="${SHARNESS_TEST_SRCDIR}/scripts/waitfile.lua"
+
+flux getattr broker.pid
+
+#  Launch some test instance jobs with varying characteristics
+#
+test_expect_success 'start a set of Flux instances' '
+	id=$(flux mini submit flux start /bin/true) &&
+	id2=$(flux mini submit -n2 -c1 flux start \
+		"flux mini run /bin/false ; \
+		 flux mini submit --cc=1-4 sleep inf && \
+		 touch ready && \
+		 flux queue idle") &&
+	flux job wait-event $id clean &&
+	$waitfile -t 60 ready
+'
+test_expect_success 'flux-jobs can get instance info' "
```

Comment:
perhaps we need some `!P` percentage tests (just check atleast `%` is output?")

Reply:
Good suggestion. I was struggling with how to make a good test for `!P`, but your idea is reasonable.